src/components/Button.js

```diff
@@ -2,14 +2,25 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const Button = props => {
-  const { name } = props;
+  const { object } = props;
+  const className = object.id % 4 === 0 || object.id === 19 ? 'operator' : 'normal';
```

Comment:
Instead of determining the zero button like this, use the wide key in the props, ![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/WideProps.png)